CHANGELOG.next.asciidoc

```diff
@@ -255,6 +255,7 @@ otherwise no tag is added. {issue}42208[42208] {pull}42403[42403]
 - Fix unexpected EOF detection and improve memory usage. {pull}44813[44813]
 - Fixed issue for "Root level readerConfig no longer respected" in azureblobstorage input. {issue}44812[44812] {pull}44873[44873]
 - Added missing "text/csv" content-type filter support in GCS input. {issue}44922[44922] {pull}44923[44923]
+- Fixed GCP Pub/Sub input integration tests  {pull}45150[45150]
```

Comment:
Because no changes from this PR are user facing (they don't need to go into the release notes), the changelog should go into `CHANGELOG-developer.next.asciidoc`.

Reply:
Got it